flink-tests/src/test/java/org/apache/flink/test/accumulators/AccumulatorLiveITCase.java

```diff
@@ -160,7 +160,7 @@ public void go() throws Exception {
 			FutureUtils.retrySuccessfulWithDelay(
 				() -> {
 					try {
-						return CompletableFuture.completedFuture(client.getAccumulators(jobGraph.getJobID()));
+						return CompletableFuture.completedFuture(client.getAccumulators(jobGraph.getJobID())).get();
```

Comment:
I think the `get()` should be in the parenthesis: 
```
CompletableFuture.completedFuture(client.getAccumulators(jobGraph.getJobID())).get()
```

Right?

Reply:
Thanks for spotting this. Correctly we should `get` on the inner future so that we get a completed future.